cross-chain/arbitrum/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@keep-network/tbtc-v2-arbitrum",
+  "version": "0.0.1-dev",
```

Comment:
Nitpicking but we could make it `1.0.0-dev`. We will soon deploy `v1.0.0` of this module.